apps/studio/components/interfaces/SQLEditor/SQLEditor.utils.ts

```diff
@@ -95,7 +95,14 @@ export function getDiffTypeDropdownLabel(diffType: DiffType) {
 }
 
 export function checkDestructiveQuery(sql: string) {
-  return destructiveSqlRegex.some((regex) => regex.test(removeCommentsFromSql(sql)))
+  const cleanedSql = removeCommentsFromSql(sql)
+  return destructiveSqlRegex.some((regex) => regex.test(cleanedSql))
+}
+
+// Function to check for UPDATE queries without WHERE clause
+export function isUpdateWithoutWhere(sql: string): boolean {
+  const updateWithoutWhereRegex = /^update\s+[\w.]+\s+set\s+[\w\W]+(?!where\s)/is
```

Comment:
just tested this on quoted identifiers and I think the regex needs to be updated to account for those: `"public"."countries"`

Reply:
great thanks!   Updated this — catches all of them now for me 

UPDATE countries SET name = 'New Name';
UPDATE public.countries SET name = 'New Name';
UPDATE "public"."countries" SET name = 'New Name';